broke-broker/src/main/java/org/dsngroup/broke/broker/handler/MqttMessageHandler.java

```diff
@@ -39,8 +39,8 @@ public void channelRead(ChannelHandlerContext ctx, Object msg) {
             MqttMessage mqttMessage = (MqttMessage) msg;
             switch (mqttMessage.fixedHeader().messageType()) {
                 case CONNECT:
-                    protocolProcessor.processConnect(ctx.channel(), (MqttConnectMessage) mqttMessage);
                     logger.debug("[MqttMessageHandler] CONNECT");
```

Comment:
In my opinion, remove `logger.debug` is better. It is not necessary for debugging here, debug when you need.

Reply:
I just want to make sure the protocol processor is keep processing the incoming message when debugging this:
In ```MultiClient``` test, the ```Server``` will eventually block in ```MessageDispatcher``` when writing ```PUBLISH``` to the subscriber.
```JAVA
matchSubscription.getSubscriberChannel()
    .writeAndFlush(mqttPublishMessageOut)
    .sync();
```

Maybe this ```logger.debug()``` message can be removed after figuring out this bug.

Reply:
Of course, this seems to be an unrelated message. It is fine to remove this now.

Reply:
Ok, merge this first.